cli/azd/resources/apphost/templates/resources.bicept

```diff
@@ -337,7 +338,45 @@ resource {{bicepName $name}} 'Microsoft.DocumentDB/databaseAccounts@2023-04-15'
     }
   }
 {{end -}}
+{{end -}}
+{{range $name, $value := .SqlServers}}
+resource {{bicepName $name}} 'Microsoft.Sql/servers@2022-05-01-preview' = {
+  name: '{{$name}}-${resourceToken}'
+  location: location
+  tags: union(tags, {'aspire-resource-name': '{{$name}}'})
+  properties: {
+    minimalTlsVersion: '1.2'
+    publicNetworkAccess: 'Enabled'
+    administrators: {
+      administratorType: 'ActiveDirectory'
+      azureADOnlyAuthentication: true
+      login: managedIdentity.name
+      principalType: 'User'
+      sid: managedIdentity.properties.principalId
+      tenantId: subscription().tenantId
+    }
+  }
 }
+
+resource {{bicepName $name}}Firewall 'Microsoft.Sql/servers/firewallRules@2022-05-01-preview' = {
+    name: 'fw-{{$name}}-${resourceToken}'
+    parent: {{bicepName $name}}
+    properties: {
+      startIpAddress: '0.0.0.0'
+      endIpAddress: '255.255.255.255'
+    }
+}
+  {{range $cname := $value.Databases}}
+  resource {{bicepName $cname}} 'Microsoft.Sql/servers/databases@2022-05-01-preview' = {
+  name: '{{$cname}}'
```

Comment:
NIT: Would you mind indenting these properties, it will allow the generated bicep to look a little nicer.